ara/clients/utils.py

```diff
@@ -20,14 +22,18 @@ def get_client(
     if username is not None and password is not None:
         auth = HTTPBasicAuth(username, password)
 
+    cert = None
+    if cert is not None and key is not None:
+        cert = (cert, key)
```

Comment:
My understanding is that they must both be set for it to work but if only one is defined, it would silently do nothing.

Could we raise an exception if either are set but not both ? It can be similar to the validation we're doing for the API client just below.

Reply:
Yes, both must be set. 

I think the exception would be better rather than silently ignoring it. I'll change that.